CI build failed on main

### Describe the bug

> error[E0432]: unresolved import `indexmap`
  --> datafusion/physical-expr/src/equivalence/class.rs:23:5
   |
23 | use indexmap::IndexSet;
   |     ^^^^^^^^ use of undeclared crate or module `indexmap`

Details: https://github.com/apache/datafusion/actions/runs/12151105350/job/33885120868

### To Reproduce

_No response_

### Expected behavior

_No response_

### Additional context

_No response_

--- a/datafusion/physical-expr/src/equivalence/class.rs
+++ b/datafusion/physical-expr/src/equivalence/class.rs
@@ -24,8 +24,7 @@ use std::fmt::Display;
 use std::sync::Arc;
 
 use datafusion_common::tree_node::{Transformed, TransformedResult, TreeNode};
-use datafusion_common::IndexSet;
-use datafusion_common::JoinType;
+use datafusion_common::{IndexSet, JoinType};
 use datafusion_physical_expr_common::physical_expr::format_physical_expr_list;
 
 /// A structure representing a expression known to be constant in a physical execution plan.

--- a/datafusion/physical-expr/src/equivalence/class.rs
+++ b/datafusion/physical-expr/src/equivalence/class.rs
@@ -20,11 +20,11 @@ use crate::{
     expressions::Column, physical_exprs_contains, LexOrdering, LexRequirement,
     PhysicalExpr, PhysicalExprRef, PhysicalSortExpr, PhysicalSortRequirement,
 };
-use indexmap::IndexSet;
 use std::fmt::Display;
 use std::sync::Arc;
 
 use datafusion_common::tree_node::{Transformed, TransformedResult, TreeNode};
+use datafusion_common::IndexSet;
 use datafusion_common::JoinType;
 use datafusion_physical_expr_common::physical_expr::format_physical_expr_list;
 

